src/nvim/getchar.c

```diff
@@ -2494,8 +2489,7 @@ fix_input_buffer (
   for (i = len; --i >= 0; ++p) {
```

Comment:
In the original patch this comment  was removed:

    // Don't replace K_SPECIAL when reading a script file.